tests/system/aiplatform/test_dataset.py

```diff
@@ -137,88 +127,35 @@ def setup_method(self):
         importlib.reload(initializer)
         importlib.reload(aiplatform)
 
-    @pytest.fixture()
-    def create_staging_bucket(self, shared_state):
-        new_staging_bucket = f"temp-sdk-integration-{uuid.uuid4()}"
-        storage_client = storage.Client()
-        storage_client.create_bucket(new_staging_bucket)
-        shared_state["storage_client"] = storage_client
-        shared_state["staging_bucket"] = new_staging_bucket
-        yield
-
-    @pytest.fixture()
-    def dataset_gapic_client(self):
-        gapic_client = dataset_service.DatasetServiceClient(
-            client_options=client_options.ClientOptions(api_endpoint=_TEST_API_ENDPOINT)
-        )
-
-        yield gapic_client
+        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
 
     @pytest.fixture()
-    def create_text_dataset(self, dataset_gapic_client, shared_state):
-
-        gapic_dataset = gca_dataset.Dataset(
-            display_name=f"temp_sdk_integration_test_create_text_dataset_{uuid.uuid4()}",
-            metadata_schema_uri=aiplatform.schema.dataset.metadata.text,
-        )
-
-        create_lro = dataset_gapic_client.create_dataset(
-            parent=_TEST_PARENT, dataset=gapic_dataset
-        )
-        new_dataset = create_lro.result()
-        shared_state["dataset_name"] = new_dataset.name
-        yield
+    def storage_client(self):
+        yield storage.Client(project=e2e_base._PROJECT)
```

Comment:
Preference to reference either  `_TEST_PROJECT`(line 130) or `e2e_base._PROJECT` but not both.